### PR TITLE
Add plugin package name to the app config

### PR DESCRIPTION
**Pull request by ipanova**
[noissue]
RequiredPR: https://github.com/pulp/pulpcore/pull/2823

### CODE REVIEW

**Review by bmbouter — APPROVED** [review]
Thanks!

File: pulp_container/app/__init__.py [hunk 1]
```diff
@@ -7,3 +7,4 @@ class PulpContainerPluginAppConfig(PulpPluginAppConfig):
     name = "pulp_container.app"
     label = "container"
     version = "2.13.0.dev"
+    python_package_name = "pulp_container"
```
Comment (gerrod3) [inline pulp_container/app/__init__.py:10]: Is this correct? Are we suppose to use the underscore or the dash version of the name. In `setup.py` the name has a dash.

Comment (bmbouter) [reply pulp_container/app/__init__.py:10]: Dang this is the one I didn't look up. It is with the `-` https://pypi.org/project/pulp-container/

**Review by bmbouter — CHANGES_REQUESTED** [review]
@ipanova it needs the dash

**Comment by ipanova** [comment]
> @ipanova it needs the dash

oops

**Pull request merged**